app/helpers/validation_helper.rb

```diff
@@ -106,7 +106,10 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     native_hawaiian_or_other_pacific_islander: { label: 'Native Hawaiian or Other Pacific Islander', checks: [:bool] },
     ethnicity: { label: 'Ethnicity', checks: [:enum] },
     interpretation_required: { label: 'Interpretation Required?', checks: [:bool] },
-    address_state: { label: 'State', checks: [:state] },
+    address_line_1: { label: 'Address 1', checks: [:required] },
+    address_city: { label: 'Town/City', checks: [:required] },
+    address_zip: { label: 'Zip', checks: [:required] },
+    address_state: { label: 'State', checks: %i[required date] },
```

Comment:
Where was the required-ness of address established? It seems it wasn't required before in import.

Reply:
Yeah, I defined "required-ness" based on what was being required in the UI. Generally that matched what would be required on import, if the defined validations were actually run, but address was the odd one out. I couldn't seem to get through enrollment without giving an address, so I made it required here.

Reply:
Cool, that sounds like a good source of truth for required-ness to me.

Reply:
Yeah the enrollment wizard is where we require fields, we don't enforce it on import yet (though we should eventually).